client/memory.cpp

```diff
@@ -30,6 +30,14 @@ using namespace RemoteCL;
 using namespace RemoteCL::Client;
 
 
+struct MappedPtr {
+	void* ptr;
```

Comment:
If the mapping owns this memory this should either be a std::unique_ptr<uint8_t[]> or std::vector<uint8_t> for automatic memory release.